src/js/jsx/search/SearchBar.jsx

```diff
@@ -396,6 +402,51 @@ define(function (require, exports, module) {
             this.refs.datalist.removeAutofillSuggestion();
             event.stopPropagation();
         },
+
+        /**
+         * Get the exact type of layer
+         *
+         * @param {string} id
+         *
+         * @return {string} layerKind
+         */
+        _findLayerType: function (id) {
```

Comment:
I really don't like this. I actually think it would be fine to NOT log the layer kind here, and instead just log that a layer was searched for (or whatever the search category is). I do not want to add a bunch of code that exists only to reverse-engineer the search IDs just for the sake of analytics.